docs/src/constrained/blockshuffle.md

```diff
@@ -0,0 +1,20 @@
+# Block shuffle surrogates
+
+Randomly shuffled surrogates are generated by dividing the original signal into 
+blocks, then permuting those blocks. Block positions are randomized, and 
+blocks at the end of the signal gets wrapped around to the start of the time 
+series.
+
+Thus, they keep short-term correlations within
+blocks, but destroy any long-term dynamical information in the signal.
```

Comment:
I am always in favor of avoiding duplication. This information should be inside the docstring, and the docstring of `BlockShuffle` expanded in the docs.


If you want to have individual pages for each method, with an example below each one, that is totally fine, but then each page should just show the docstring instead of duplicating info.

Duplication makes everything harder to maintain.